wordpress-mobile/WordPress-iOS
# Prepare "Classes" for conversion to folders

Prerequisite for #23629 – we can't adopt folders until we drop CocoaPods, but all of the hard work can be done in advance.

This PR aligns the file system and the project.

To Test:
- Ensure this PR builds in CI
- Open the project in Xcode, right-click on the "Classes" directory, hold Option, then click `Preflight Conversion to Folder`. Note that it says `“Classes” is ready to be converted to a folder.`
